src/current/v25.2/create-policy.md

```diff
@@ -0,0 +1,106 @@
+---
+title: CREATE POLICY
+summary: The CREATE POLICY statement defines a new row-level security policy on a table.
+toc: true
+keywords: security, row level security, RLS
+docs_area: reference.sql
+---
+
+The `CREATE POLICY` statement defines a new [row-level security (RLS)]({% link {{ page.version.version }}/row-level-security.md %}) policy on a [table]({% link {{ page.version.version }}/schema-design-table.md %}).
+
+## Syntax
+
+<!--
+
+NB. This was waiting on a fix to DOC-12125 when this doc was being
+written. Now there is additional followup work (tracked in DOC-13653)
+to update the parameters and potentially the diagram.
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/create_policy.html %}
+</div>
+
+-->
+
+{% include_cached copy-clipboard.html %}
+~~~
+CREATE POLICY [ IF NOT EXISTS ] policy_name ON table_name
+    [ AS ( PERMISSIVE | RESTRICTIVE ) ]
+    [ FOR { ALL | SELECT | INSERT | UPDATE | DELETE } ]
```

Comment:
{ } here too and in the params table

Reply:
gosh you're right, must have missed these - you're def in sync

edited the "choice" brackets here and in the params table to be parens instead

also did the same on alter-policy.md to change brackets to be parens